subprojects/core/src/main/java/org/gradle/internal/service/scopes/VirtualFileSystemServices.java

```diff
@@ -99,11 +99,9 @@
     public static final String VFS_PARTIAL_INVALIDATION_ENABLED_PROPERTY = "org.gradle.unsafe.vfs.partial-invalidation";
 
     /**
-     * Boolean system property to enable retaining VFS state between builds.
+     * The old system property used to enable watching the file system.
      *
-     * Also enables partial VFS invalidation.
-     *
-     * @see #VFS_PARTIAL_INVALIDATION_ENABLED_PROPERTY
+     * Using this property causes Gradle to emit a deprecation warning.
      */
```

Comment:
I guess we could add `@Deprecated` here, so that we'll find this property while removing deprecations in 7.0.

Reply:
I think we should remove the property already in 6.6.